.github/workflows/prettier.yml

```
@@ -0,0 +1,31 @@
+name: Prettier
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: lint
+      id: prettier-run
+      uses: rutajdash/prettier-cli-action@v1.0.0
```

I think we can use so, that we are always on v1 latest
```suggestion
      uses: rutajdash/prettier-cli-action@v1
```